django/test/client.py

```diff
@@ -39,9 +39,8 @@
 
 
 class RedirectCycleError(Exception):
-    """
-    The test client has been asked to follow a redirect loop.
-    """
+    """The test client has been asked to follow a redirect loop."""
+
```

Comment:
I'd skip the blank line after the class docstring changes. Focus on changing the verbs only.